.concierge/templates/pullRequestOpened.hbs

```diff
@@ -1,4 +1,14 @@
 {{#if claEnabled}}
+{{#if errorCla}}
+@{{ userName }}, thanks for the pull request!
+
+If this is your first contribution, can you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request?
```

Comment:
I would remove the `can you` and change the question mark into a period.  Otherwise I think the sentence flow is weird.